cmd/policy-loader/main.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2015 ISRG.  All rights reserved
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package main
+
```

Comment:
I propose a style guide: for our standalone cmds, let's put the "what it does" in a multiline usage statement, so it gets echoed if you run the command. Then if we move `main()` to the top of the file, it also serves something you can read quickly to orient yourself when reading the source.